### PR TITLE
add an option '--comment' to assign local ASS comment file, or /dev/null to disable comment.

### PR DESCRIPTION
**Pull request by persmule**
It may be useful that bilidan has the ability to use local converted ASS comment file with parsed remote media, or to disable downloading and displaying comments when users want so. So I managed to achieve this.

### CODE REVIEW

File: bilidan.py [hunk 1]
```diff
@@ -300,13 +300,18 @@ def launch_player(video_metadata, media_urls, comment_out, is_playlist=False, in
         video_size = (1920, 1080)
 
     logging.info('Loading comments...')
-    comment_out = convert_comments(video_metadata['cid'], video_size)
+    if comment is None:
+        comment_out = convert_comments(video_metadata['cid'], video_size)
+    else:
+        comment_out = open(comment, 'r')
+        comment_out.close()
```
Comment (m13253) [inline bilidan.py]: Why closing?

File: bilidan.py [hunk 2]
```diff
@@ -300,13 +300,18 @@ def launch_player(video_metadata, media_urls, comment_out, is_playlist=False, in
...
 
     logging.info('Launching media player...')
     player_exit_code = launch_player(video_metadata, media_urls, comment_out, increase_fps=not keep_fps)
 
-    if player_exit_code == 0:
-        os.remove(comment_out.name)
+    if comment is None:
+        if player_exit_code == 0:
```
Comment (m13253) [inline bilidan.py]: Consider merging into
```py
if comment is None and player_exit_code == 0:
```

File: bilidan.py (hunk 1 above)
Comment (persmule) [inline bilidan.py]: Because convert_comments closed before return, and only its path is actually used.

File: bilidan.py (hunk 2 above)
Comment (persmule) [inline bilidan.py]: Okay.

File: bilidan.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+1#!/usr/bin/env python3
```
Comment (m13253) [inline bilidan.py]: Why are you changing the shebang? This will cause this program unable to execute.

Comment (persmule) [inline bilidan.py]: Sorry, it is a typo. I was so hurry to edit the file.

**Comment by m13253** [comment]
Actually, you can always press 'V' to toggle comments when playing.

**Comment by persmule** [comment]
I know. But these days xml comments downloadings fails frequently, so I guess I may be able to add an option to disable comment before downloading it, and I noticed this function can be extended to what I have implemented.

**Comment by m13253** [comment]
Okay, thank you! :smiley:

**Pull request merged**